src/transactions/strategy.js

```diff
@@ -0,0 +1,66 @@
+import { Fixed8 } from '../utils'
+
+/**
+ * Select the coins in order of value, smallest first.
+ * @param {AssetBalance} assetBalance
+ * @param {Fixed8} requiredAmt
+ * @return {Coin[]}
+ */
+export const smallestFirst = (assetBalance, requiredAmt) => {
+  // Ascending order sort
+  assetBalance.unspent.sort((a, b) => a.value.sub(b.value).toNumber())
+  return fillFromLeft(requiredAmt, assetBalance.unspent)
+}
+
+/**
+ * Select the coins in order of value, biggest first.
+ * @param {AssetBalance} assetBalance
+ * @param {Fixed8} requiredAmt
+ * @return {Coin[]}
+ */
+export const biggestFirst = (assetBalance, requiredAmt) => {
+  // Descending order sort
+  assetBalance.unspent.sort((a, b) => b.value.sub(a.value).toNumber())
+  return fillFromLeft(requiredAmt, assetBalance.unspent)
+}
+
+/**
+ * Use a balanced approach by selecting a coin that

```

Comment:
@snowypowers This comment appears incomplete.  Can you describe how the balanced strategy is supposed to work compared to the others defined here?  Thanks!

Reply:
Found the documentation [here](https://cityofzion.io/neon-js/docs/en/api/core/tx.html#calculation-strategies).  Might be nice to update the comment since it got cut off. 😄